src/styles/Blocks/cards.scss

```diff
@@ -6,6 +6,7 @@ $cards-vertical-padding: 40px;
 .cards {
   display: flex;
   flex-wrap: wrap;
+  margin: 0 auto;
```

Comment:
let's move this property to .main__cards, because you can't set the positioning within the block itself, the positioning must be set relative to the parent block separately (according to BEM)

Reply:
Agree, fixed it